dashboards-observability/opensearch_dashboards.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "observabilityDashboards",
-  "version": "2.0.0.0",
+  "version": "2.0.0.0-alpha1",
   "opensearchDashboardsVersion": "2.0.0",
```

Comment:
What does this correspond to?

Reply:
plugin version and compatible dashboards version